staking/programs/staking/src/state/global_config.rs

```diff
@@ -13,6 +13,12 @@ pub struct GlobalConfig {
     pub unlocking_duration:    u8,
     pub epoch_duration:        u64, // epoch duration in seconds
     pub freeze:                bool,
+
+    /// Once the pyth token is listed, governance can update the config to set this value.
+    /// Once this value is set, vesting schedules that depend on the token list date can start vesting.
+    // FIXME: do options serialize correctly? iirc there was some issue about this
+    pub pyth_token_list_time:       Option<i64>,
```

Comment:
This field is a little problematic because governance can claw back already vested funds by turning this from Some to None. Not sure how a better implementation would work tho.

Reply:
Yeah. i don't see a way around this, especially since governance can also upgrade the program.